feature/flags.go

```diff
@@ -51,3 +51,7 @@ const DisableRaft = "disable-raft"
 
 // UpgradeSeries is a development feature flag.
 const UpgradeSeries = "upgrade-series"
+
+// RaftLeases will switch all lease management to be handled by the
+// raft lease FSM, rather than being done in Mongo.
+const RaftLeases = "raft-leases"
```

Comment:
This is on by default in 2.5 right?

Reply:
It wasn't, but after we talked about it I'll change it to be on by default. In that case I should really flip the sense of the flag to be "disable-raft-leases"

Reply:
Better suggestion (from @wallyworld): `use-legacy-leases`.